livox_ros2_driver/livox_ros2_driver/lddc.cpp

```diff
@@ -718,427 +708,461 @@ void Lddc::onDiagnosticsTimer()
   for (uint32_t i = 0; i < kMaxSourceLidar; ++i) {
     if (lds_->lidars_[i].handle != kMaxSourceLidar) ++lidar_count_;
   }
+
+  if (connected_lidar_count_ != lds_->connected_lidars_.size())
+  {
+    connected_lidar_count_ = lds_->connected_lidars_.size();
+    for (const auto &lidar : lds_->connected_lidars_)
+    {
+      const std::string broadcast_code = lidar.first;
+
+      updater_.add(
+          fmt::format("livox_temperature_{}", broadcast_code),
```

Comment:
By the way, it might be better to use ` {broadcast_code}` or `-{broadcast_code}` instead of `_{broadcast_code}` considering diagnostic_aggregator.

Currently, we filter diagnostics by name like `: livox_temperature`.
So if we have two diagnostics `livox_temperature` and `livox_temperature_change`, we can't filter correctly.
Of course, we can avoid this problem by using regex.

Reply:
Some diagnostics use ` `(whitespace) for the delimiter, so `-` is better.
But this is not necessary.

Reply:
https://github.com/tier4/livox_ros2_driver/pull/14/commits/bd8b0185d14e2b81a6130b6db420c32f3c002e51

Reply:
You mean if we have `livox_temperature-bdcode` and `livox_temperature_change-bdcode` we can filter `livox_temperature-bdcode` by `livox_temperature-` right?

Reply:
Yes, exactly! (but anyway it's hacky and not beautiful. lol)